docs/security/content_trust.md

```diff
@@ -0,0 +1,156 @@
+Enter passphrase for new root key with id 3a97f:
+Repeat passphrase for new root key with id 3a97f:
+Enter passphrase for new targets key with id docker.io/diogomonica/alpine/03cf0:
+Repeat passphrase for new targets key with id docker.io/diogomonica/alpine/03cf0:
+Finished initializing "docker.io/diogomonica/alpine"
+```
+
+It is **very important** to note that loss of the root key will require contacting Docker's support team to reset the repository state and will also require **manual intervention** from every docker user that has used the repository before (removing the old root certificate). You should backup this key somewhere safe. We will show you how to backup your keys later on in this document.
+
+Additionally, the passphrase you chose for both the root key and your content key-pair should be randomly generated and stored in a **password manager**.  You can find more details on how these keys are used in the content trust key structure section.
+
+Now that we have pushed the tag `latest`, we can confirm that our trusted pull now works as expected:
+
+```
+➜  docker pull diogomonica/alpine
+Using default tag: latest
+Pull (1 of 1): diogomonica/alpine:latest@sha256:d149ab53f871
+...
+Tagging diogomonica/alpine@sha256:d149ab53f871 as diogomonica/alpine:latest
+```
+
+#### Building with trusted content
+
+Now lets try to build the simple Dockerfile below. Since we are using `FROM diogomonica/alpine:latest`, we are expecting this build to succeed, given that trust data exists for the tag `latest`.
+
+```
+➜  cat Dockerfile
+FROM diogomonica/alpine:latest
+RUN echo
+➜  docker build -t diogomonica/alpine:testing .
+Using default tag: latest
+latest: Pulling from diogomonica/alpine
+
+b3dbab3810fc: Pull complete
+a9539b34a6ab: Pull complete
+Digest: sha256:d149ab53f871
+```
+
+If we try to build from a Dockerfile that relied on tag without trust data, the build command will fail:
+
+```
+➜  cat Dockerfile
+FROM diogomonica/alpine:notrust
+RUN echo
+➜  docker build -t diogomonica/alpine:testing .
+unable to process Dockerfile: No trust data for notrust
+```
+
+#### Doing untrusted operations
+
+A user that wants to disable trust for a particular operation can use the `--disable-content-trust` flag. **Warning: this flag disables content trust for this operation**. With this flag, Docker will ignore content-trust and allow all operations to be done without verifying any signatures. If we wanted the previous untrusted build to succeed we could do:
+
+```
+➜  cat Dockerfile
+FROM diogomonica/alpine:notrust
+RUN echo
+➜  docker build --disable-content-trust -t diogomonica/alpine:testing .
+Sending build context to Docker daemon 42.84 MB
+...
+Successfully built f21b872447dc
+```
+
+The same is true for all the other commands, such as `pull` and `push`:
+
+```
+➜  docker pull --disable-content-trust diogomonica/alpine:untrusted
+...
+➜  docker push --disable-content-trust diogomonica/alpine:untrusted
+...
+```
+
```

Comment:
how can a user see if an image on their Docker host ignored content-trust, or had it set? 

can you show an example of the `docker inspect` command (if that will do it) - will `docker images` show this information?

(the user's daemon could well be restarted with the option on or off, so showing the state is useful)

also - do we show the derived trust of a container based on its image?